README.md

```diff
@@ -207,6 +207,46 @@ consult the Solr logs to confirm the core was remade.
 Nice! The production server should have the newly edited schema and freshly
 built index, ready to search, filter, and facet.
 
+## Adding a new board member
+
+Hooray! A new member has been elected or appointed to the Board of Directors.
+Metro will provide a headshot and bio for the new member. There are a few
+changes you need to make so they appear correctly on the site.
+
+1. Add the new member to the `VOTING_POSTS` object in the Metro person scraper,
+then merge and [follow the steps to deploy your change](https://github.com/datamade/scrapers-us-municipal/#deploying-changes).
+    - Example: https://github.com/opencivicdata/scrapers-us-municipal/pull/337
+    - Tip: Run `docker-compose run --rm scrapers pupa import lametro people --rpm=0` to capture the change locally.
+2. After the revised person scrape runs, remove any board memberships for the
+new member that were created without a post.
+    ```python
+    from lametro.models import Person
+    Person.objects.get(family_name='<MEMBER LAST NAME>').memberships.filter(organization__name='Board of Directors', post__isnull=True).delete()
+    ```
+3. Add the new member's headshot to the `lametro/static/images/manual-headshots`
+directory, then update the `MANUAL_HEADSHOTS` object in `councilmatic/settings_jurisdiction.py`. **Be sure to key the headshot off the person slug
+from the production site.**
+    - Example: https://github.com/datamade/la-metro-councilmatic/pull/686
+4. Add the new member's bio to the `MEMBER_BIOS` object in `councilmatic/settings_jurisdiction.py`, again **using the person slug from production.**
+    - Example: https://github.com/datamade/la-metro-councilmatic/pull/686
+    - Tip: Replace newlines in the provided bio with `<br /><br />`.
```

Comment:
We talked about this briefly, but re-organizing this so that it's clear that steps 1 & 2 don't have to happen before steps 3 & 4 would be helpful.

Additionally, linking to our instructions on [CI deployment using tags](https://github.com/datamade/deploy-a-site/blob/8efe413f18383caa0cb5c81373f817afd6a944d7/How-to-deploy-with-continuous-deployment.md#3-deploy-to-production) would be helpful

Reply:
Revised instructions. Added the link to docs on tags to the deployment instructions in our scraper fork. https://github.com/datamade/scrapers-us-municipal/#deploying-changes